src/views/viewTabs.component.tsx

```diff
@@ -44,7 +44,7 @@ function a11yProps(label: TabValue) {
   };
 }
 
-const StyledTab = styled(Tab)(({ theme }) => ({
+export const StyledTab = styled(Tab)(({ theme }) => ({
```

Comment:
Since we're reusing the `StyledTab` code, it might be worthwhile trying to reuse the other code above - would just need to use generics to type `TabValue` I think